openapi.yml

```diff
@@ -42,15 +42,14 @@ x-commons:
       $ref: https://teamdigitale.github.io/openapi/0.0.5/definitions.yaml#/responses/429TooManyRequests
     503:
       $ref: https://teamdigitale.github.io/openapi/0.0.5/definitions.yaml#/responses/503ServiceUnavailable
-    default:
```

Comment:
LGTM @gunzip `io-utils` both swagger and oas3 can't use `default` as a schema name. Is `default` a reserved typescript keyword?

Reply:
it is a reserved keywords in javascript (but it may be not related, it depends on the exact error you get using io-utils)